index.ts

```diff
@@ -26,14 +26,21 @@ export const background = rootContext();
 export const TODO = rootContext();
 
 export const requestIdKey = Symbol("requestId");
-
 export function getRequestId(ctx: Context): string | undefined {
   if (requestIdKey in ctx) {
     // typescript doesn't like symbols, but it's safe
     return String((ctx as Record<typeof requestIdKey, unknown>)[requestIdKey]);
   }
 }
 
+export const actorKey = Symbol("actor");
+export function getActor(ctx: Context): string | undefined {
+  if (actorKey in ctx) {
+    // typescript doesn't like symbols, but it's safe
+    return String((ctx as Record<typeof actorKey, unknown>)[actorKey]);
+  }
+}
```

Comment:
Just bringing up for conversation whether this is where it should live - is this something that is more to RPC, vs everything?

I'm thinking everything as we probably set this on the request handler from the internet, eg Office

Reply:
Needs to be defined in both the RPC client and server, since we already have the request ID in here it seemed like a suitable place. It definitely spreads out the logic a lot though